NBitcoin/Secp256k1/ECPubKey.cs

```diff
@@ -304,6 +305,29 @@ private bool secp256k1_eckey_pubkey_tweak_add(ECMultContext ctx, ref GE key, in
 			return true;
 		}
 
+		/// <summary>
+		/// The original function name is `secp256k1_ec_pubkey_combine`
+		/// </summary>
+		public static bool TryCombine(Context ctx, IEnumerable<ECPubKey> pubkeys, out ECPubKey? combinedPubKey)
+		{
+			if (pubkeys == null)
+				throw new ArgumentNullException(nameof(pubkeys));
+
+			var count = pubkeys.Count();
```

Comment:
It is an `IEnumerable`, I would prefer you do not cause it to be evaluated twice.